pkg/controller/manager/sync_test.go

```diff
@@ -305,6 +305,118 @@ func TestManagerSynchronization(t *testing.T) {
 
 			Actions: []action{&deleteTaskAction{clusterID: clusterID, taskID: "other-repair-id"}},
 		},
+		{
+			Name: "Special 'now' startDate is not compared during update decision",
+			Spec: scyllav1.ScyllaClusterSpec{
+				Repairs: []scyllav1.RepairTaskSpec{
+					{
+						SchedulerTaskSpec: scyllav1.SchedulerTaskSpec{
+							Name:      "repair",
+							StartDate: "now",
+							Interval:  "0",
+						},
+						Intensity:           "666",
+						SmallTableThreshold: "1GiB",
+					},
+				},
+			},
+			Status: scyllav1.ScyllaClusterStatus{
+				ManagerID: pointer.StringPtr(clusterID),
+				Repairs: []scyllav1.RepairTaskStatus{
+					{
+						ID: "repair-id",
+						RepairTaskSpec: scyllav1.RepairTaskSpec{
+							SchedulerTaskSpec: scyllav1.SchedulerTaskSpec{
+								Name:      "repair",
+								StartDate: "2021-01-01T11:11:11Z",
+								Interval:  "0",
+							},
+							Intensity:           "666",
+							SmallTableThreshold: "1GiB",
+						},
+					},
+				},
+			},
+			State: state{
+				Clusters: []*mermaidclient.Cluster{{
+					ID:        clusterID,
+					Name:      clusterName,
+					AuthToken: clusterAuthToken,
+				}},
+				RepairTasks: []*RepairTask{
+					{
+						RepairTaskSpec: scyllav1.RepairTaskSpec{
+							SchedulerTaskSpec: scyllav1.SchedulerTaskSpec{
+								Name:      "repair",
+								StartDate: "2021-01-01T11:11:11Z",
+								Interval:  "0",
+							},
+							Intensity:           "666",
+							SmallTableThreshold: "1GiB",
+						},
+						ID: "repair-id",
+					},
+				},
+			},
+
+			Actions: nil,
+		},
+		{
+			Name: "Update task when startDate is changed",
```

Comment:
nit: It'd be better to make the tests' names grammatically concise, i.e. if the first test's name is "Special 'now' startDate is not compared during update decision", the other should also say e.g. "Task gets updated when startDate is changed".

Reply:
done